src/GapsInCareHelpers.ts

```diff
@@ -30,11 +30,11 @@ export function processQueriesForGaps(
   return queries.map(q => {
     // Determine satisfaction of parent query and leaf node retrieve
     const parentQueryResult = detailedResult.clauseResults?.find(
-      cr => cr.libraryName === q.libraryName && cr.localId === q.queryLocalId
+      cr => cr.libraryName === q.retrieveLibraryName && cr.localId === q.queryLocalId
```

Comment:
```suggestion
      cr => cr.libraryName === q.queryLibraryName && cr.localId === q.queryLocalId
```
This one wants the query results.